src/test/js/login_user_functional_test.js

```diff
@@ -15,6 +15,7 @@ const testSuitePrefix = randomData.getRandomAlphabeticString();
 const serviceName =  randomData.getRandomServiceName(testSuitePrefix);
 const serviceClientSecret = randomData.getRandomClientSecret();
 const userPassword = randomData.getRandomUserPassword();
+const appendedStringPromise = randomData.getRandomTextFor11KB();
```

Comment:
can this be a more descriptive name? "largeCookiePromise"? (also, why is it a promise? not sure why the get value needs to be async)

Reply:
Added promise to address the protocol error on the cookie value , now after moving to text generation to glolabl declaration  i can use it without asynch